### PR TITLE
Bump node to 20.10.0, Bump lint staged to 15.1.0

### PR DESCRIPTION
**Pull request by 0b5vr**
This supercedes #1330 

This PR bumps lint-staged to 15.1.0.

lint-staged v15 requires node 18.2.0 at least, this PR also bumps node to 20.9.0 (LTS latest) as well.

See: https://github.com/lint-staged/lint-staged/releases/tag/v15.0.0

Also, the previous installation of lint-staged was not working correctly, making a lot of git stash and fails, sometimes deleting the staged / unstaged changes without commit.
This PR should resolve the behavior.

### CODE REVIEW

File: .node-version [hunk 1]
```diff
@@ -1 +1 @@
-16.17.1
+20.9.0
```
Comment (yue4u) [inline .node-version]: https://nodejs.org/en/blog/release/v20.10.0 (LTS) just released so maybe consider 20.10.0 too. Keep it as 20.9.0 is also ok.

File: packages/three-vrm-core/package.json [hunk 2]
```diff
@@ -52,7 +52,6 @@
   },
   "devDependencies": {
     "@types/three": "^0.154.0",
-    "lint-staged": "13.1.2",
```
Comment (yue4u) [inline packages/three-vrm-core/package.json:55]: Root lint-staged seems to work with per-package "lint-staged" config but currently html/js files aka `lint-examples` is not included. Is this intended?

File: .node-version (hunk 1 above)
Comment (0b5vr) [reply .node-version]: gonna bump to 20.10.0

File: packages/three-vrm-core/package.json (hunk 2 above)
Comment (0b5vr) [reply packages/three-vrm-core/package.json:55]: `lint-examples` should also be handled by lint-staged, indeed! Will do in a separated PR.

File: .node-version (hunk 1 above)
Comment (0b5vr) [reply .node-version]: addressed @ https://github.com/pixiv/three-vrm/pull/1332/commits/3a2e3a06e63e228abb7f285e011d3ffbbb82fc84

**Review by yue4u — APPROVED** [review]

**Pull request merged**